src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.product-container .stars--1 .stars__star:first-child,
+.product-container .stars--2 .stars__star:nth-child(-n+2),
+.product-container .stars--3 .stars__star:nth-child(-n+3),
+.product-container .stars--4 .stars__star:nth-child(-n+4),
+.product-container .stars--5 .stars__star:nth-child(-n+5) {
```

Така велика вкладеність непотрібна
```suggestion
.stars--1 .stars__star:first-child,
```